### PR TITLE
Preptile

### LINKED ISSUE #94: run the desmeds prep code also
Now that we are committing to storing the info using pizza cutter code, we might as well also run the prep code too

### PR DESCRIPTION
**Pull request by esheldon**
closes #94

### CODE REVIEW

File: pizza_cutter/des_pizza_cutter/_des_info.py [hunk 1]
```diff
@@ -74,24 +63,6 @@ def get_des_coadd_tile_info(
...
-        medsconf,
-        tilename,
-        band,
-        campaign=campaign,
-        sources=coadd_srcs,
-    )
-
-    info = coadd.get_info()
-    info['tilename'] = tilename
-    info['band'] = band
```
Comment (beckermr) [inline pizza_cutter/des_pizza_cutter/_des_info.py]: The tilename and band info is used in other parts of the code. We should either take that out or add it back.

File: bin/des-pizza-cutter-prep-tile [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    prep = desmeds.desdm_maker.Preparator(
+        cfg,
+        args.tilename,
+        args.band,
+    )
+    if args.clean:
+        prep.clean()
+
+    else:
+        prep.go()
```
Comment (beckermr) [inline bin/des-pizza-cutter-prep-tile]: Is it this call that does the download?

File: bin/des-pizza-cutter-prep-tile [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    config_text = jinja2.Template(_config).render(
+        tilename=args.tilename,
+        band=args.band,
+    )
+    cfg = yaml.load(config_text, Loader=yaml.Loader)
+
+    cfg['medsconf'] = os.path.basename(args.config).replace('.yaml', '')
+
+    prep = desmeds.desdm_maker.Preparator(
+        cfg,
```
Comment (beckermr) [inline bin/des-pizza-cutter-prep-tile]: Which entries in the config file is the `Preparator` using?

**Comment by beckermr** [comment]
Right now we have a lot of processing on the data products that we may want to change when making the MEDS files. Is the prep code downloading the SE images etc to a place that is versioned under the medsconf, or some other place? These should be separated out. This has been on my mental list, but I have not done it yet.

File: pizza_cutter/des_pizza_cutter/_des_info.py (hunk 1 above)
Comment (esheldon) [reply pizza_cutter/des_pizza_cutter/_des_info.py]: that info is already in the info dict, so no need to add it again

File: bin/des-pizza-cutter-prep-tile (hunk 2 above)
Comment (esheldon) [reply bin/des-pizza-cutter-prep-tile]: yep

Comment (beckermr) [reply bin/des-pizza-cutter-prep-tile]: We should add a comment then.

**Comment by esheldon** [comment]
that's fine; it uses the "medsconf" to determine where to put the data, we can make that something set in the config file rather than determined from the config file name.

Note that we will most likely be running `--clean` after we make the MEDS files so this will not be permanent storage.  In that case this doesn't matter much.  But I agree if we plan to keep data around we should separate it.

File: bin/des-pizza-cutter-prep-tile (hunk 3 above)
Comment (esheldon) [reply bin/des-pizza-cutter-prep-tile]: I added a comment

File: bin/des-pizza-cutter-prep-tile (hunk 2 above)
Comment (esheldon) [reply bin/des-pizza-cutter-prep-tile]: comments added

File: bin/des-pizza-cutter-prep-tile [hunk 4]
```diff
@@ -64,6 +64,7 @@ def main():
 
     cfg['medsconf'] = os.path.basename(args.config).replace('.yaml', '')
 
+    # Preparator uses campaign, source_type from config
```
Comment (beckermr) [inline bin/des-pizza-cutter-prep-tile]: Does it also use medsconf?

Comment (esheldon) [reply bin/des-pizza-cutter-prep-tile]: yep

**Comment by beckermr** [comment]
Duplicated output:

```tile info file: /Users/Matt/DESDATA/y3v02/pizza_cutter_info/DES2122+0001_i_pizza_cutter_info.yaml
dumping tile information: /Users/Matt/DESDATA/y3v02/pizza_cutter_info/DES2122+0001_i_pizza_cutter_info.yaml```

**Comment by esheldon** [comment]
ready to merge?

**Comment by beckermr** [comment]
yup let's go for it!

**Pull request merged**